docs/administrator/configuration/configure-controllers.md

```diff
@@ -166,17 +167,15 @@ More details please refer to:
 - [bootstrap tokens overview](https://kubernetes.io/docs/reference/access-authn-authz/bootstrap-tokens/#bootstrap-tokens-overview)
 - [enabling bootstrap token authentication](https://kubernetes.io/docs/reference/access-authn-authz/bootstrap-tokens/#enabling-bootstrap-token-authentication)
 
-#### csrapproving, csrcleaner, csrsigning
+#### csrcleaner, csrsigning
 
 The controllers runs as part of `kube-controller-manager`.
 
-The `csrapproving` controller uses the [SubjectAccessReview API](https://kubernetes.io/docs/reference/access-authn-authz/authorization/#checking-api-access) to determine if a given user is authorized to request a CSR, then approves based on the authorization outcome.
-
 The `csrcleaner` controller clears expired csr periodically.
 
 The `csrsigning` controller signs the certificate using Karmada root CA.
 
-> Note: these controllers currently are used to register member clusters with PULL mode by `karmadactl register`.
+> Note: these controllers currently operate with the `agentcsrapproving` controller to register member clusters in PULL mode using `karmadactl register`.
```

Comment:
I'm afraid this would bring ambiguous as there is no `agentcsrapproving` within kube-controller-manager.

Reply:
How about 
```suggestion
> Note: these controllers currently operate with the `agentcsrapproving` controller enabled by karmada-controller-manager to register member clusters in PULL mode using `karmadactl register`.
```

Reply:
much better